src/cli.ts

```diff
@@ -63,10 +65,12 @@ if (strategy !== 'highest' && strategy !== 'fewer') {
 
 try {
     const yarnLock = fs.readFileSync(file, 'utf8');
+    const packageJson = packageJsonPath ? fs.readFileSync(packageJsonPath, 'utf8') : undefined;
```

Comment:
Nit: I rather use `null` instead of `undefined` in this case. It feels more explicit to me.